File: Labs/lab03/generatekey.py
import random 

def random_number(min, max=None):
    if max is None:
        max = min
        min = 0
    number = int(random.random() * (max - min + 1)) + min
    return number

def generate_key(length, characters):
    full_length = len(characters)
    key = ''

    for _ in range(length):
        key += characters[random_number(0, full_length - 1)]

    return key

characters = "abcdefghijklmnopqrstuvwxyz0123456789"
key = generate_key(15, characters)
print(key)

File: Labs/lab03/generatekey.js
function random_namber(min, max=null) {
    let actual_min, actual_max;
    
    if (max === null){
        actual_min = 0;
        actual_max = min;
     } else {
        actual_min = min;
        actual_max = max;
     }
        
    const number = Math.floor(Math.random() * (actual_max - actual_min + 1)) + actual_min
    return number
}

function generate_key(length, characters){
    const full_length = characters.length
    let key = '';
    for (let i = 0; i<length; i++ ) {
        key += characters[random_namber(0,  full_length-1)]
    }
    return key
}

const characters = "abcdefghijklmnopqrstuvwxyz0123456789";
const key = generate_key(16, characters);

console.log(key)